staging/src/k8s.io/kubelet/pkg/apis/deviceplugin/v1alpha/api.pb.go

```diff
@@ -259,12 +259,17 @@ func (m *Device) GetHealth() string {
 	return ""
```

Comment:
revert the changes to this file (I suspect you need to update your dev environment to go1.19rc2, see https://groups.google.com/a/kubernetes.io/g/dev/c/Z7NLSTRVE0Y)